app/services/catalog_record_id.rb

```diff
@@ -43,10 +43,14 @@ def type
     def previous_type
       "previous #{type}"
     end
+
+    def other_type
+      'symphony'
+    end
   end
 
   def self.links(model)
-    new(model).links
+    new(model).links.map(&:catalogRecordId)
```

Comment:
i found it a little confusing that this class method `links` does the `.map(&:catalogRecordId)`, but instance method `links` doesn't.

Reply:
ah, i see that the same is true for the instance and class method versions of `other_links`, so at least it's consistently inconsistent 🙂, and keeps from breaking an interface with the rest of the codebase, so that makes sense too.

also weird, now that i think about it, that some class methods are defined using the `class << self` approach, and others the `self.` approach, but that's not something i'd ask to see changed in this PR, since this PR didn't introduce that.